dist/api_data.py

```diff
@@ -597,6 +597,9 @@ def __ge__(self, other):
             if true, reallocation of memory will only provide the exact amount requested. This
             will help with spotting memory allocation issues more easily.''',
             type='boolean'),
+        Config('realloc_malloc', 'false', r'''
+            if true, malloc will be used to allocate memory and mimic realloc function.''',
```

Comment:
Hmm, I thought that it was that every realloc call would force a new memory allocation. That is not quite what this says.